root/scripts/gamemodes.txt

```diff
@@ -106,7 +106,7 @@
 			// get set on 360 and dedicated for 360
 			"360_z_mega_mob_size" 25
 			"360_z_mob_spawn_max_size" 18
-			"360_z_versus_wandering_density" 0.026
+			"360_versus_wandering_zombie_density" 0.026
```

Comment:
Assuming that the desired cvar is not `z_wandering_density`, which even has a farther default `0.03` as opposed to `0.027`; and that the real 360 cvar is not incorrect, as in the typo, thus following the simple prepending of `360_`.